js-exercises/alphabetic-shift/alphabeticShift.js

```diff
@@ -0,0 +1,7 @@
+const alphabeticShift = input => {
+  return input.toString().split("").map(char => char === "z" ? "a" : char === "Z" ? "A" : String.fromCharCode(char.charCodeAt() + 1)).join("");
```

Comment:
This is correct but hard to understand easily. Maybe you can split it into more lines. Too much is happening in line. Just my opinion, so I am not sure of it. Also, you can do it the way Arfat taught in lectures. I really liked his solution to rot13. It was easy, readable and reusable.

Reply:
Thanks, next time I will split my code.